conf/scalardb-custom-values.yaml

```diff
@@ -0,0 +1,128 @@
+envoy:
```

Comment:
Modify the custom file with the configuration that the user needs to change.

Reply:
Okay, I will check and confirm and update the configuration file accordingly.

Reply:
Updated custom file with configurations with data that is required by user to change.

Reply:
Okay, I will update the file by adding the above

Reply:
Updated as specified.